step.yml

```diff
@@ -266,16 +266,6 @@ inputs:
       - xcpretty
       - xcodebuild
       is_required: true
-  - is_export_xcarchive_zip: "no"
-    opts:
-      category: Debug
-      title: Export the generated archive?
-      description: |-
-        If this input is set to `yes`, the generated .xcarchive will be zipped and moved to `output_dir`.
```

Comment:
Please remove line 337 " This environment will be exported only if, is_export_xcarchive_zip="yes""